CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## PENDING
 
+Features
+* [cosmos-sdk-cli] Added support for cosmos-sdk-cli under cosmos-sdk/cmd
+  This allows SDK users to init a new project repository with a single command.
+=======
```

Comment:
changes should be under the existing `FEATURES` header (also we haven't been using `======` dividers)